pkg/harness/stackrun/v1/types.go

```diff
@@ -60,6 +63,8 @@ func (in *StackRun) FromStackRunBaseFragment(fragment *gqlclient.StackRunBaseFra
 		Variables:    fragment.Variables,
 		PolicyEngine: fragment.PolicyEngine,
 		DryRun:       fragment.DryRun,
+		Parallelism:  fragment.Configuration.Terraform.Parallelism,
```

Comment:
Configuration.Terraform can be nil and this will segfault.  It's better to just keep that entire struct as a field and nil check internally for safety